src/sequelize/models/User.js

```diff
@@ -80,6 +80,7 @@ export default (sequelize, DataTypes) => {
     User.hasMany(models.Reaction, { foreignKey: 'userId', as: 'LikerOrDisliker', onDelete: 'CASCADE' });
     User.hasMany(models.Comment, { foreignKey: 'userId', as: 'Commenter', onDelete: 'CASCADE' });
     User.hasMany(models.Bookmark, { foreignKey: 'userId', onDelete: 'CASCADE' });
+    User.hasMany(models.Highlight, { foreignKey: 'userId', onDelete: 'CASCADE' });
```

Comment:
I wonder why you didn't give alias name this association? But otherwise looks good.